tests/stress/stress_test.go

```diff
@@ -0,0 +1,107 @@
+// Copyright 2020 ChainSafe Systems (ON) Corp.
+// This file is part of gossamer.
+//
+// The gossamer library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The gossamer library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the gossamer library. If not, see <http://www.gnu.org/licenses/>.
+
+package rpc
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strconv"
+	"testing"
+
+	"github.com/ChainSafe/gossamer/dot/rpc/modules"
+	"github.com/ChainSafe/gossamer/tests/rpc"
+	"github.com/stretchr/testify/require"
+
+	scribble "github.com/nanobox-io/golang-scribble"
+)
+
+var (
+	pidList = make([]*exec.Cmd, 3)
+)
+
+func TestMain(m *testing.M) {
+	if rpc.GOSSAMER_INTEGRATION_TEST_MODE != "stress" {
+		_, _ = fmt.Fprintln(os.Stdout, "Going to skip stress test")
+		return
+	}
+
+	_, _ = fmt.Fprintln(os.Stdout, "Going to start stress test")
```

Comment:
are the `_, _` needed here?

Reply:
yes, output need to be explicitly ignored.

Reply:
is this a lint error? could you add a comment to explain why they need to be explicitly ignored?

Reply:
This is like this everywhere else, lets focus on the fundamentals not on the details.